src/cc/frontends/clang/loader.cc

```diff
@@ -108,11 +109,24 @@ int ClangLoader::parse(unique_ptr<llvm::Module> *mod, TableStorage &ts,
   unique_ptr<llvm::MemoryBuffer> main_buf;
   struct utsname un;
   uname(&un);
-  string kdir = string(KERNEL_MODULES_DIR) + "/" + un.release;
-  auto kernel_path_info = get_kernel_path_info (kdir);
+  string kdir, kpath;
+  const char *kpath_env = ::getenv("BCC_KERNEL_SOURCE");
```

Comment:
Again about env variable vs. cmake macro? I would prefer cmake macro to make the code less dependent on external factors. If use cmake macro, maybe KERNEL_SOURCE_DIR as the macro name?

Reply:
Same reason as above, BCC should adapt to different kernel sources that the same BCC installation can point to dynamically. We shouldn't restrict this to a statically known path.

Reply:
Okay with me for this.